Travis CI does not run anymore

The Travis CI build does not run anymore for recent PRs, such as e.g. #567.

The README points to the old wrong TravisCI.org instead of TravisCI.com which is now used.

https://github.com/openMF/mobile-wallet/commits/dev has links to CloudBees Codeship, e.g. https://app.codeship.com/projects/332704/builds/b17ee1e9-ffe3-4c9f-98ea-1dc2cafcf059, but those do not appear to be public?  Also the README only links to and mentions TravisCI, not Codeship.

I'll see if I can do anything quick & easy to make TravisCI kick in again...

@edcable @Bhavnaharitsa @jsahil730 @luckyman20 FYI, feel free to chime in here, if anything

The `master` branch currently [still passes the build on Travis](https://travis-ci.com/openMF/mobile-wallet/builds/138872442).

But the `dev` branch of this project currently does not build anymore (on Travis), see https://travis-ci.com/openMF/mobile-wallet/builds/138872225, due to `Caused by: org.gradle.api.GradleException: Checkstyle rule violations were found. See the report at: file:///home/travis/build/openMF/mobile-wallet/mifospay/build/reports/checkstyle/checkstyle.html`

PS: https://github.com/openMF/mobile-wallet/blob/dev_redesign/.travis.yml makes it look like this project at least at some point in the past definitely used Travis... if this project meanwhile would like to use CloudBees Codeship instead of Travis, then that file should be removed and the README updated accordingly... otherwise could I suggest that whoever set up CloudBees Codeship disable that again?

--- a/config/quality/pmd/pmd-ruleset.xml
+++ b/config/quality/pmd/pmd-ruleset.xml
@@ -1,35 +1,37 @@
 <?xml version="1.0"?>
 <ruleset xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" name="Android Application Rules"
-         xmlns="http://pmd.sf.net/ruleset/1.0.0"
-         xsi:noNamespaceSchemaLocation="http://pmd.sf.net/ruleset_xml_schema.xsd"
-         xsi:schemaLocation="http://pmd.sf.net/ruleset/1.0.0 http://pmd.sf.net/ruleset_xml_schema.xsd">
+         xmlns="http://pmd.sourceforge.net/ruleset/2.0.0"
+         xsi:schemaLocation="http://pmd.sourceforge.net/ruleset/2.0.0
+          https://pmd.sourceforge.io/ruleset_2_0_0.xsd">
 
-    <description>Custom ruleset for Mifos Self-Service Android application</description>
+    <description>Custom ruleset for Mifos Pay Android application</description>
 
     <exclude-pattern>.*/R.java</exclude-pattern>
     <exclude-pattern>.*/gen/.*</exclude-pattern>
 
     <rule ref="rulesets/java/android.xml" />
+    <rule ref="rulesets/java/basic.xml" >
+        <exclude name="AvoidUsingHardCodedIP" />
+    </rule>
+    <rule ref="rulesets/java/braces.xml">
+        <!-- We allow single line if's without braces -->
+        <exclude name="IfStmtsMustUseBraces" />
+    </rule>
     <rule ref="rulesets/java/clone.xml" />
     <rule ref="rulesets/java/finalizers.xml" />
     <rule ref="rulesets/java/imports.xml">
         <!-- Espresso is designed this way !-->
         <exclude name="TooManyStaticImports" />
     </rule>
-    <rule ref="rulesets/java/logging-java.xml">
-        <!-- This rule wasn't working properly and given errors in every var call info -->
-        <exclude name="GuardLogStatementJavaUtil" />
-    </rule>
-    <rule ref="rulesets/java/braces.xml">
-        <!-- We allow single line if's without braces -->
-        <exclude name="IfStmtsMustUseBraces" />
-    </rule>
-    <rule ref="rulesets/java/strings.xml" >
-        <!-- Exclude because causes problems with SQL Strings that usually require duplication -->
-        <exclude name="AvoidDuplicateLiterals"/>
-    </rule>
-    <rule ref="rulesets/java/basic.xml" >
-        <exclude name="AvoidUsingHardCodedIP" />
+    <rule ref="category/java/codestyle.xml/ClassNamingConventions">
+        <properties>
+            <property name="classPattern" value="[A-Z][a-zA-Z0-9]*" />
+            <property name="abstractClassPattern" value="[A-Z][a-zA-Z0-9]*" />
+            <property name="interfacePattern" value="[A-Z][a-zA-Z0-9]*" />
+            <property name="enumPattern" value="[A-Z][a-zA-Z0-9]*" />
+            <property name="annotationPattern" value="[A-Z][a-zA-Z0-9]*" />
+            <property name="utilityClassPattern" value="[A-Z][a-zA-Z0-9]*" />
+        </properties>
     </rule>
     <rule ref="rulesets/java/naming.xml">
         <exclude name="AbstractNaming" />
@@ -38,5 +40,10 @@
         <exclude name="ShortVariable" />
         <exclude name="ShortClassName" />
         <exclude name="VariableNamingConventions" />
+        <exclude name="ClassNamingConventions" />
+    </rule>
+    <rule ref="rulesets/java/strings.xml" >
+        <!-- Exclude because causes problems with SQL Strings that usually require duplication -->
+        <exclude name="AvoidDuplicateLiterals"/>
     </rule>
 </ruleset>

--- a/core/src/main/java/org/mifos/mobilewallet/core/base/UseCaseThreadPoolScheduler.java
+++ b/core/src/main/java/org/mifos/mobilewallet/core/base/UseCaseThreadPoolScheduler.java
@@ -2,7 +2,6 @@ package org.mifos.mobilewallet.core.base;
 
 import android.os.Handler;
 
-import java.util.concurrent.ArrayBlockingQueue;
 import java.util.concurrent.Executors;
 import java.util.concurrent.ThreadPoolExecutor;
 import java.util.concurrent.TimeUnit;

--- a/core/src/main/java/org/mifos/mobilewallet/core/data/fineract/api/services/SavingsAccountsService.java
+++ b/core/src/main/java/org/mifos/mobilewallet/core/data/fineract/api/services/SavingsAccountsService.java
@@ -6,7 +6,6 @@ import org.mifos.mobilewallet.core.data.fineract.entity.Page;
 import org.mifos.mobilewallet.core.data.fineract.entity.accounts.savings.SavingAccount;
 import org.mifos.mobilewallet.core.data.fineract.entity.accounts.savings.SavingsWithAssociations;
 import org.mifos.mobilewallet.core.data.fineract.entity.accounts.savings.Transactions;
-import org.mifos.mobilewallet.core.domain.model.Transaction;
 
 import retrofit2.http.Body;
 import retrofit2.http.GET;
@@ -34,7 +33,8 @@ public interface SavingsAccountsService {
             @Path("accountId") long accountId,
             @Query("command") String command);
 
-    @GET(ApiEndPoints.SAVINGS_ACCOUNTS + "/{accountId}/" + ApiEndPoints.TRANSACTIONS + "/{transactionId}")
+    @GET(ApiEndPoints.SAVINGS_ACCOUNTS + "/{accountId}/" + ApiEndPoints.TRANSACTIONS
+            + "/{transactionId}")
     Observable<Transactions> getSavingAccountTransaction(@Path("accountId") long accountId,
                                                          @Path("transactionId") long transactionId);
 }

--- a/core/src/main/java/org/mifos/mobilewallet/core/data/fineract/repository/FineractRepository.java
+++ b/core/src/main/java/org/mifos/mobilewallet/core/data/fineract/repository/FineractRepository.java
@@ -240,7 +240,8 @@ public class FineractRepository {
                         Constants.TRANSACTIONS);
     }
 
-    public Observable<Transactions> getSelfAccountTransactionFromId(long accountId, long transactionId) {
+    public Observable<Transactions> getSelfAccountTransactionFromId(long accountId,
+                                                                    long transactionId) {
         return selfApiManager
                 .getSavingAccountsListApi().getSavingAccountTransaction(accountId,
                         transactionId);

--- a/core/src/main/java/org/mifos/mobilewallet/core/domain/usecase/account/DownloadTransactionReceipt.java
+++ b/core/src/main/java/org/mifos/mobilewallet/core/domain/usecase/account/DownloadTransactionReceipt.java
@@ -16,7 +16,8 @@ import rx.schedulers.Schedulers;
  */
 
 public class DownloadTransactionReceipt extends
-        UseCase<DownloadTransactionReceipt.RequestValues, DownloadTransactionReceipt.ResponseValue> {
+        UseCase<DownloadTransactionReceipt.RequestValues,
+                DownloadTransactionReceipt.ResponseValue> {
 
     private final FineractRepository mFineractRepository;
 

--- a/core/src/main/java/org/mifos/mobilewallet/core/domain/usecase/account/FetchAccountTransaction.java
+++ b/core/src/main/java/org/mifos/mobilewallet/core/domain/usecase/account/FetchAccountTransaction.java
@@ -34,7 +34,8 @@ public class FetchAccountTransaction extends UseCase<FetchAccountTransaction.Req
     @Override
     protected void executeUseCase(final FetchAccountTransaction.RequestValues requestValues) {
 
-        fineractRepository.getSelfAccountTransactionFromId(requestValues.accountId, requestValues.transactionId)
+        fineractRepository.getSelfAccountTransactionFromId(requestValues.accountId,
+                requestValues.transactionId)
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribeOn(Schedulers.io())
                 .subscribe(new Observer<Transactions>() {
@@ -68,8 +69,13 @@ public class FetchAccountTransaction extends UseCase<FetchAccountTransaction.Req
             this.accountId = accountId;
         }
 
-        public void setAccountId(long accountId) { this.accountId = accountId; }
-        public void setTransactionId(long transactionId) {this.transactionId = transactionId;}
+        public void setAccountId(long accountId) {
+            this.accountId = accountId;
+        }
+
+        public void setTransactionId(long transactionId) {
+            this.transactionId = transactionId;
+        }
     }
 
     public static final class ResponseValue implements UseCase.ResponseValue {

--- a/core/src/main/java/org/mifos/mobilewallet/core/domain/usecase/invoice/FetchInvoice.java
+++ b/core/src/main/java/org/mifos/mobilewallet/core/domain/usecase/invoice/FetchInvoice.java
@@ -1,7 +1,6 @@
 package org.mifos.mobilewallet.core.domain.usecase.invoice;
 
 import android.net.Uri;
-import android.widget.Toast;
 
 import org.mifos.mobilewallet.core.base.UseCase;
 import org.mifos.mobilewallet.core.data.fineract.entity.Invoice;
@@ -63,7 +62,7 @@ public class FetchInvoice extends UseCase<FetchInvoice.RequestValues, FetchInvoi
                             }
                         }
                     });
-        } catch(IndexOutOfBoundsException e) {
+        } catch (IndexOutOfBoundsException e) {
             getUseCaseCallback().onError("Invalid link used to open the App");
         }
 

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -3,4 +3,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-all.zip

--- a/mifospay/build.gradle
+++ b/mifospay/build.gradle
@@ -75,6 +75,8 @@ dependencies {
     implementation "io.reactivex:rxandroid:$rootProject.rxandroidVersion"
     implementation "io.reactivex:rxjava:$rootProject.rxjavaVersion"
 
+    implementation "io.michaelrocks:libphonenumber-android:8.11.0"
+
     implementation 'me.dm7.barcodescanner:zxing:1.8.4'
 
     implementation 'com.mifos.mobile:mifos-passcode:0.3.0'

--- a/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/data/local/PreferencesHelper.java
+++ b/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/data/local/PreferencesHelper.java
@@ -133,9 +133,13 @@ public class PreferencesHelper {
         putString(CLIENT_VPA, vpa);
     }
 
-    public void setAccountId(long accountId) {putLong(ACCOUNT_ID,accountId);}
+    public void setAccountId(long accountId) {
+        putLong(ACCOUNT_ID, accountId);
+    }
 
-    public Long getAccountId() {return getLong(ACCOUNT_ID,0);}
+    public Long getAccountId() {
+        return getLong(ACCOUNT_ID, 0);
+    }
 
     public String getFirebaseRegId() {
         return getString(FIREBASE_REG_ID, "");

--- a/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/BaseHomeContract.java
+++ b/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/BaseHomeContract.java
@@ -6,7 +6,6 @@ import org.mifos.mobilewallet.core.domain.model.client.Client;
 import org.mifos.mobilewallet.mifospay.base.BasePresenter;
 import org.mifos.mobilewallet.mifospay.base.BaseView;
 
-import java.util.ArrayList;
 import java.util.List;
 
 import okhttp3.ResponseBody;

--- a/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/presenter/HomePresenter.java
+++ b/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/home/presenter/HomePresenter.java
@@ -15,7 +15,6 @@ import org.mifos.mobilewallet.mifospay.history.TransactionsHistory;
 import org.mifos.mobilewallet.mifospay.home.BaseHomeContract;
 import org.mifos.mobilewallet.mifospay.utils.Constants;
 
-import java.util.ArrayList;
 import java.util.List;
 
 import javax.inject.Inject;
@@ -44,7 +43,8 @@ public class HomePresenter implements BaseHomeContract.HomePresenter,
     private List<Transaction> transactionList;
 
     @Inject
-    public HomePresenter(UseCaseHandler useCaseHandler, LocalRepository localRepository, PreferencesHelper preferencesHelper) {
+    public HomePresenter(UseCaseHandler useCaseHandler, LocalRepository localRepository,
+                         PreferencesHelper preferencesHelper) {
         this.mUsecaseHandler = useCaseHandler;
         this.localRepository = localRepository;
         this.preferencesHelper = preferencesHelper;
@@ -94,7 +94,8 @@ public class HomePresenter implements BaseHomeContract.HomePresenter,
     private void handleTransactionsHistory(int existingItemCount) {
         int transactionsAmount = transactionList.size() - existingItemCount;
         if (transactionsAmount > Constants.HOME_HISTORY_TRANSACTIONS_LIMIT) {
-            List<Transaction> showList = transactionList.subList(0, Constants.HOME_HISTORY_TRANSACTIONS_LIMIT + existingItemCount);
+            List<Transaction> showList = transactionList.subList(0,
+                    Constants.HOME_HISTORY_TRANSACTIONS_LIMIT + existingItemCount);
             mHomeView.showTransactionsHistory(showList);
             mHomeView.showBottomSheetActionButton();
         } else {

--- a/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/merchants/adapter/MerchantsAdapter.java
+++ b/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/merchants/adapter/MerchantsAdapter.java
@@ -94,7 +94,9 @@ public class MerchantsAdapter extends RecyclerView.Adapter<MerchantsAdapter.View
         };
     }
 
-    public List<SavingsWithAssociations> getMerchants() {return mMerchantsList;}
+    public List<SavingsWithAssociations> getMerchants() {
+        return mMerchantsList;
+    }
 
     public class ViewHolder extends RecyclerView.ViewHolder {
 

--- a/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/merchants/presenter/MerchantTransferPresenter.java
+++ b/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/merchants/presenter/MerchantTransferPresenter.java
@@ -29,7 +29,7 @@ import static org.mifos.mobilewallet.core.utils.Constants.FETCH_ACCOUNT_TRANSFER
  */
 
 public class MerchantTransferPresenter implements BaseHomeContract.MerchantTransferPresenter,
-        HistoryContract.TransactionsHistoryAsync{
+        HistoryContract.TransactionsHistoryAsync {
 
     private final UseCaseHandler mUsecaseHandler;
     private final LocalRepository localRepository;
@@ -51,7 +51,9 @@ public class MerchantTransferPresenter implements BaseHomeContract.MerchantTrans
 
 
     @Inject
-    public MerchantTransferPresenter(UseCaseHandler useCaseHandler, LocalRepository localRepository, PreferencesHelper preferencesHelper) {
+    public MerchantTransferPresenter(UseCaseHandler useCaseHandler,
+                                     LocalRepository localRepository,
+                                     PreferencesHelper preferencesHelper) {
         this.mUsecaseHandler = useCaseHandler;
         this.localRepository = localRepository;
         this.preferencesHelper = preferencesHelper;

--- a/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/merchants/ui/MerchantTransferActivity.java
+++ b/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/merchants/ui/MerchantTransferActivity.java
@@ -36,7 +36,8 @@ import static org.mifos.mobilewallet.mifospay.MifosPayApp.getContext;
  * Created by Shivansh Tiwari on 06/07/19.
  */
 
-public class MerchantTransferActivity extends BaseActivity implements BaseHomeContract.MerchantTransferView {
+public class MerchantTransferActivity extends BaseActivity implements
+        BaseHomeContract.MerchantTransferView {
 
     private BottomSheetBehavior mBottomSheetBehavior;
 
@@ -96,7 +97,8 @@ public class MerchantTransferActivity extends BaseActivity implements BaseHomeCo
         TextDrawable drawable = TextDrawable.builder().beginConfig()
                 .width((int) getResources().getDimension(R.dimen.user_profile_image_size))
                 .height((int) getResources().getDimension(R.dimen.user_profile_image_size))
-                .endConfig().buildRound(getIntent().getStringExtra(Constants.MERCHANT_NAME).substring(0, 1), R.color.colorPrimary);
+                .endConfig().buildRound(getIntent().getStringExtra(Constants.MERCHANT_NAME)
+                        .substring(0, 1), R.color.colorPrimary);
         ivMerchantImage.setImageDrawable(drawable);
         showTransactionFetching();
         setUpRecycleView();
@@ -153,7 +155,7 @@ public class MerchantTransferActivity extends BaseActivity implements BaseHomeCo
     @Override
     public void showPaymentDetails(String externalId, double amount) {
         MakeTransferFragment fragment = MakeTransferFragment.newInstance(externalId, amount);
-        fragment.show(getSupportFragmentManager(),"tag");
+        fragment.show(getSupportFragmentManager(), "tag");
     }
 
     @Override

--- a/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/merchants/ui/MerchantsFragment.java
+++ b/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/merchants/ui/MerchantsFragment.java
@@ -98,12 +98,17 @@ public class MerchantsFragment extends BaseFragment implements MerchantsContract
                                 .get(position).getExternalId();
                         if (merchantVPA == null) {
                             Toast.makeText(getActivity(),
-                                    "VPA is Null. Can't make any transactions.",Toast.LENGTH_LONG).show();
+                                    "VPA is Null. Can't make any transactions.",
+                                    Toast.LENGTH_LONG).show();
                         } else {
-                            Intent intent = new Intent(getActivity(), MerchantTransferActivity.class);
-                            intent.putExtra(Constants.MERCHANT_NAME,mMerchantsAdapter.getMerchants().get(position).getClientName());
-                            intent.putExtra(Constants.MERCHANT_VPA,mMerchantsAdapter.getMerchants().get(position).getExternalId());
-                            intent.putExtra(Constants.MERCHANT_ACCOUNT_NO,mMerchantsAdapter.getMerchants().get(position).getAccountNo());
+                            Intent intent = new Intent(getActivity(),
+                                    MerchantTransferActivity.class);
+                            intent.putExtra(Constants.MERCHANT_NAME, mMerchantsAdapter
+                                    .getMerchants().get(position).getClientName());
+                            intent.putExtra(Constants.MERCHANT_VPA, mMerchantsAdapter
+                                    .getMerchants().get(position).getExternalId());
+                            intent.putExtra(Constants.MERCHANT_ACCOUNT_NO, mMerchantsAdapter
+                                    .getMerchants().get(position).getAccountNo());
                             startActivity(intent);
                         }
                     }
@@ -116,12 +121,13 @@ public class MerchantsFragment extends BaseFragment implements MerchantsContract
                                 .get(position).getExternalId();
                         if (merchantVPA == null) {
                             Toast.makeText(getActivity(),
-                                    "VPA is Null, can't be copied.",Toast.LENGTH_LONG).show();
+                                    "VPA is Null, can't be copied.", Toast.LENGTH_LONG).show();
                         } else {
-                            ClipData clip = ClipData.newPlainText("VPA",merchantVPA);
+                            ClipData clip = ClipData.newPlainText("VPA", merchantVPA);
                             clipboard.setPrimaryClip(clip);
                             Toast.makeText(getActivity(),
-                                    "VPA copied to Clipboard Successfully",Toast.LENGTH_LONG).show();
+                                    "VPA copied to Clipboard Successfully",
+                                    Toast.LENGTH_LONG).show();
                         }
                     }
                 }));

--- a/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/payments/ui/SendFragment.java
+++ b/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/payments/ui/SendFragment.java
@@ -11,7 +11,6 @@ import android.os.Bundle;
 import android.provider.ContactsContract;
 import android.support.annotation.NonNull;
 import android.support.annotation.Nullable;
-import android.support.constraint.ConstraintLayout;
 import android.support.design.chip.Chip;
 import android.support.design.widget.TextInputLayout;
 import android.support.transition.TransitionManager;
@@ -127,7 +126,8 @@ public class SendFragment extends BaseFragment implements BaseHomeContract.Trans
     public void transferClicked() {
         String externalId = etVpa.getText().toString().trim();
         String eamount = etAmount.getText().toString().trim();
-        String mobileNumber = mEtMobileNumber.getText().toString().trim().replaceAll("\\s+", "");
+        String mobileNumber = mEtMobileNumber.getText()
+                .toString().trim().replaceAll("\\s+", "");
         if (eamount.equals("") || (externalId.equals("") && mobileNumber.equals(""))) {
             Toast.makeText(getActivity(),
                     Constants.PLEASE_ENTER_ALL_THE_FIELDS, Toast.LENGTH_SHORT).show();
@@ -195,7 +195,7 @@ public class SendFragment extends BaseFragment implements BaseHomeContract.Trans
         if (requestCode == SCAN_QR_REQUEST_CODE && resultCode == Activity.RESULT_OK) {
             String qrData = data.getStringExtra(Constants.QR_DATA);
             final String[] qrDataArray = qrData.split(", ");
-            if(qrDataArray.length == 1) {
+            if (qrDataArray.length == 1) {
                 etVpa.setText(qrDataArray[0]);
             } else {
                 etVpa.setText(qrDataArray[0]);

--- a/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/qr/ui/ShowQrActivity.java
+++ b/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/qr/ui/ShowQrActivity.java
@@ -24,7 +24,6 @@ import javax.inject.Inject;
 
 import butterknife.BindView;
 import butterknife.ButterKnife;
-import butterknife.OnClick;
 
 /**
  * Created by naman on 8/7/17.
@@ -95,8 +94,8 @@ public class ShowQrActivity extends BaseActivity implements QrContract.ShowQrVie
                     return;
                 }
                 mAmount = edittext.getText().toString();
-                tvQrData.setText(qrData + ", "+mAmount);
-                generateQR(qrData + ", "+mAmount);
+                tvQrData.setText(qrData + ", " + mAmount);
+                generateQR(qrData + ", " + mAmount);
             }
         });
         editTextDialog.setNeutralButton("Reset", new DialogInterface.OnClickListener() {
@@ -122,7 +121,7 @@ public class ShowQrActivity extends BaseActivity implements QrContract.ShowQrVie
     }
 
     void showToast(String message) {
-        Toast.makeText(this,message,Toast.LENGTH_LONG).show();
+        Toast.makeText(this, message, Toast.LENGTH_LONG).show();
     }
 
     void generateQR(String qrData) {

--- a/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java
+++ b/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/receipt/ui/ReceiptActivity.java
@@ -104,7 +104,7 @@ public class ReceiptActivity extends BaseActivity implements ReceiptContract.Rec
                         return true;
                     }
                 });
-            } catch(IndexOutOfBoundsException e) {
+            } catch (IndexOutOfBoundsException e) {
                 showToast("Invalid link used to open the App.");
             }
             showProgressDialog(Constants.PLEASE_WAIT);
@@ -148,9 +148,11 @@ public class ReceiptActivity extends BaseActivity implements ReceiptContract.Rec
             tvPaidToName.setText(transferDetail.getFromClient().getDisplayName());
         }
         tvTransToName.setText(Constants.NAME + transferDetail.getToClient().getDisplayName());
-        tvTransToNumber.setText(Constants.ACCOUNT_NUMBER + transferDetail.getToAccount().getAccountNo());
+        tvTransToNumber.setText(Constants.ACCOUNT_NUMBER + transferDetail
+                .getToAccount().getAccountNo());
         tvTransFromName.setText(Constants.NAME + transferDetail.getFromClient().getDisplayName());
-        tvTransFromNumber.setText(Constants.ACCOUNT_NUMBER + transferDetail.getFromAccount().getAccountNo());
+        tvTransFromNumber.setText(Constants.ACCOUNT_NUMBER + transferDetail
+                .getFromAccount().getAccountNo());
         hideProgressDialog();
     }
 

--- a/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/registration/presenter/SignupPresenter.java
+++ b/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/registration/presenter/SignupPresenter.java
@@ -74,7 +74,8 @@ public class SignupPresenter implements RegistrationContract.SignupPresenter {
     @Override
     public void checkPasswordStrength(String password) {
         PasswordStrength p = new PasswordStrength(password);
-        mSignupView.updatePasswordStrength(p.getStrengthStringId(),p.getColorResId(),p.getValue());
+        mSignupView.updatePasswordStrength(p.getStrengthStringId(),
+                p.getColorResId(), p.getValue());
     }
 
     @Override

--- a/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/registration/ui/SignupActivity.java
+++ b/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/registration/ui/SignupActivity.java
@@ -3,7 +3,6 @@ package org.mifos.mobilewallet.mifospay.registration.ui;
 import static org.mifos.mobilewallet.mifospay.utils.FileUtils.readJson;
 
 import android.content.Intent;
-import android.graphics.Color;
 import android.net.Uri;
 import android.os.Bundle;
 import android.support.design.widget.FloatingActionButton;
@@ -211,7 +210,7 @@ public class SignupActivity extends BaseActivity implements RegistrationContract
 
         if (mifosSavingProductId
                 == org.mifos.mobilewallet.core.utils.Constants.MIFOS_MERCHANT_SAVINGS_PRODUCT_ID
-            && isEmpty(mEtBusinessShopName)) {
+                && isEmpty(mEtBusinessShopName)) {
             Toaster.showToast(this, "All fields are mandatory");
             hideProgressDialog();
             return;
@@ -224,7 +223,7 @@ public class SignupActivity extends BaseActivity implements RegistrationContract
             hideProgressDialog();
             return;
         }
-        if (mEtPassword.getText().toString().length()<6) {
+        if (mEtPassword.getText().toString().length() < 6) {
             showToast("Password should contain more than 6 characters");
             return;
         }

--- a/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/savedcards/ui/AddCardDialog.java
+++ b/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/savedcards/ui/AddCardDialog.java
@@ -131,8 +131,8 @@ public class AddCardDialog extends BottomSheetDialogFragment {
             etCardNumber.setText(editCard.getCardNumber());
             String[] expiryDate = editCard.getExpiryDate().split("/");
             int currentYear = Calendar.getInstance().get(Calendar.YEAR);
-            spnMM.setSelection(Integer.parseInt(expiryDate[0])-1);
-            spnYY.setSelection(Integer.parseInt(expiryDate[1])-currentYear);
+            spnMM.setSelection(Integer.parseInt(expiryDate[0]) - 1);
+            spnYY.setSelection(Integer.parseInt(expiryDate[1]) - currentYear);
             btnAdd.setText(Constants.UPDATE);
         }
     }

--- a/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/Constants.java
+++ b/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/Constants.java
@@ -72,7 +72,8 @@ public class Constants {
     public static final String ERROR_FINDING_VPA = "Error finding Virtual Payment Address";
     public static final String PLEASE_ENTER_VALID_AMOUNT = "Please enter a valid amount";
     public static final String SELF_ACCOUNT_ERROR = "Self Account transfer is not allowed";
-    public static final String PLEASE_ENTER_AMOUNT = "Please enter a valid amount before making the transfer";
+    public static final String PLEASE_ENTER_AMOUNT =
+            "Please enter a valid amount before making the transfer";
     public static final String NEED_READ_CONTACTS_PERMISSION = "Need read contacts permission";
     public static final String NEED_CAMERA_PERMISSION_TO_SCAN_QR_CODE =
             "Need camera permission to scan qr code.";
@@ -112,15 +113,15 @@ public class Constants {
     public static final String MERCHANT = "Merchant";
     public static final String INVOICE = "Invoice";
     public static final String NEED_EXTERNAL_STORAGE_PERMISSION_TO_DOWNLOAD_RECEIPT =
-            "Need external storage permission to download receipt.";
-    public static final String RECEIPT_DOWNLOADED_SUCCESSFULLY = "Receipt Downloaded Successfully.";
+            "Need external storage permission to download receipt";
+    public static final String RECEIPT_DOWNLOADED_SUCCESSFULLY = "Receipt Downloaded Successfully";
     public static final String ERROR_DOWNLOADING_RECEIPT = "Error downloading receipt";
     public static final String MIFOSPAY = "mifospay";
     public static final String RECEIPT = "Receipt";
     public static final String PDF = ".pdf";
     public static final String INVOICES = "Invoices";
     public static final String ERROR_FETCHING_INVOICES = "Error fetching invoices";
-    public static final String ERROR_FETCHING_RECEIPT = "Error fetching receipt.";
+    public static final String ERROR_FETCHING_RECEIPT = "Error fetching receipt";
     public static final String INVOICE_DOMAIN = "https://invoice.mifospay.com/";
 
     public static final String SENDING_OTP_TO_YOUR_MOBILE_NUMBER =

--- a/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/PasswordStrength.java
+++ b/mifospay/src/main/java/org/mifos/mobilewallet/mifospay/utils/PasswordStrength.java
@@ -14,7 +14,7 @@ public class PasswordStrength {
 
     public PasswordStrength(String password) {
 
-        Log.e ("log","INIT : "+currentScore+" "+password.length());
+        Log.e ("log", "INIT : " + currentScore + " " + password.length());
         if (password.length() < 6) {
             currentScore = 0;
         } else if (password.length() < 12) {
@@ -23,8 +23,8 @@ public class PasswordStrength {
             currentScore = 2;
         }
 
-        if (password.length()>6) {
-            for (int i=0;i<password.length();i++) {
+        if (password.length() > 6) {
+            for (int i = 0; i < password.length(); i++) {
                 char c = password.charAt(i);
 
                 if (Character.isUpperCase(c)) {
@@ -33,8 +33,8 @@ public class PasswordStrength {
                 }
             }
         }
-        if (password.length()>6) {
-            for (int i=0;i<password.length();i++) {
+        if (password.length() > 6) {
+            for (int i = 0; i < password.length(); i++) {
                 char c = password.charAt(i);
 
                 if (Character.isDigit(c)) {
@@ -43,7 +43,7 @@ public class PasswordStrength {
                 }
             }
         }
-        Log.e("log","Final : "+currentScore);
+        Log.e("log", "Final : " + currentScore);
 
         switch (currentScore) {
             case (0) : value = 0;

